### PR TITLE
Add hashbrown HashMap/HashSet implementations for Input/Output type.

### PR DESCRIPTION
**Pull request by BratSinot**
Greetings!

I don't know if that make any sense or don't look like third wheel, but `hashbrow` one in one replacement for `HashMap` from std and I using it in my project.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -33,6 +33,7 @@ async-stream = "0.3.0"
 async-trait = "0.1.48"
 fnv = "1.0.7"
 futures-util = { version = "0.3.0", default-features = false, features = ["io", "sink"] }
+hashbrown = { version="0.11.2", optional = true }
```
Comment (sunli829) [inline Cargo.toml:36]: Move this line after `# Feature optional dependencies`.

**Review by sunli829 — COMMENTED** [review]
Thank you very much! 🙂

It would be nice if you could help to add `hashbrown` feature to the `Crate Features` paragraph in `lib.rs` and `README.md`, and update `CHANGLOG`.

File: Cargo.toml (hunk 1 above)
Comment (BratSinot) [reply Cargo.toml]: Also fixed rustfmt error (always forget to enable in my IDE for new project =) ).

**Comment by sunli829** [comment]
Thanks!

**Pull request merged**